### PR TITLE
Update stringtable.xml

### PR DESCRIPTION
**Pull request by Legolasindar**
Updated spanish terms, on moved english to first raws.

### CODE REVIEW

**Comment by Legolasindar** [comment]
esteldunedain think I've done it again as before. I do the same thing to the AGM, and not how to do it differently, if you can explain exactly the steps and buttons? Thanks

**Comment by PabstMirror** [comment]
In the upper left it should say patch-3.  That's the branch your on and the branch here that you've made a pull request for.
Just keep making new commits to that branch and they will showup here automatically.

**Comment by Legolasindar** [comment]
If i select patch-3, this show my lastet merge but this cant allow me to continue with the new modification.

**Comment by PabstMirror** [comment]
Could you post a screenshot of the problem?
You should be able to make additional changes and make additional commits.

**Comment by Legolasindar** [comment]
First the Github show this.
http://i.imgur.com/iqQdKJM.png
But if i select patch-3 in the head fork, show this, and Create pull request dissapear.
http://i.imgur.com/Lebot5S.png

**Comment by esteldunedain** [comment]
Oh, so you are doing all the editing directly on the github page (browser)?? You are not using any desktop program to make the changes and upload them?

**Comment by PabstMirror** [comment]
If so, then goto here:  https://github.com/Legolasindar/ACE3/tree/patch-3

and keep making commits to that branch

**Comment by Legolasindar** [comment]
Yes, I do not use the program, downloaded it once, but I did not like his performance.

**Comment by esteldunedain** [comment]
Then do as pabst saids. All those commits will appear on this thread automatically. Let us know when you have finished and we'll merge them all in one go.

**Comment by esteldunedain** [comment]
> If i use this patch-3 link, this updated with all changues automatically?
Yes, it should

**Comment by esteldunedain** [comment]
Excellent @Legolasindar, it's working

**Comment by Legolasindar** [comment]
Thank you for helping me :)

File: addons/common/stringtable.xml [hunk 1]
```diff
@@ -296,54 +296,64 @@
...
       <English>Set the desired location of the progress bar on your screen.</English>
+      <Spanish>Seleccionar la ubicación deseada de la barra de progreso en tu pantalla</Spanish>
     </Key>
     <Key ID="STR_ACE_Common_SettingDisplayTextColorName">
       <English>Hint Background color</English>
+      <Spanish>Color de fondo de las indicaciones</Spanish>
     </Key>
     <Key ID="STR_ACE_Common_SettingDisplayTextColorDesc">
       <English>The color of the background from the ACE hints.</English>
+      <Spanish>El color de fondo de las indicaciones del ACE</Spanish>
```
Comment (esteldunedain) [inline addons/common/stringtable.xml]: These are settings, so they should be something like: "Color de fondo de las notificaciones"

Comment (Legolasindar) [inline addons/common/stringtable.xml]: Thanks, "Notificación" is much better.

**Comment by Legolasindar** [comment]
Only rest for translate realisticnames

File: addons/medical/stringtable.xml [hunk 2]
```diff
@@ -208,74 +216,97 @@
...
         <English>Elastic Bandage</English>
+        <Spanish>Vendaje elástico</Spanish>
       </Key>
       <Key ID="STR_ACE_MEDICAL_ACTIONS_QuikClot">
         <English>QuikClot</English>
+        <Spanish>QuikClot</Spanish>
       </Key>
       <Key ID="STR_ACE_MEDICAL_ACTIONS_CheckPulse">
         <English>Check Pulse</English>
+        <Spanish>Comprovar pulso</Spanish>
```
Comment (esteldunedain) [inline addons/medical/stringtable.xml]: Comprobar

Comment (Legolasindar) [inline addons/medical/stringtable.xml]: Fixed, thanks

**Comment by esteldunedain** [comment]
Thanks @Legolasindar. Please hold on a few days until we finished merging the other languages before adding more entries. Due to the way git works (and our lack of previous planning), merging all the different PRs from different languages together is becoming a pain.

**Comment by Legolasindar** [comment]
Okay, I understand :)

**Comment by Legolasindar** [comment]
My patch-3 is outdated, i have problems for maintain patch-3 at day, please now use my master branch.
https://github.com/Legolasindar/ACE3

Thanks

**Pull request merged**